README.md

```diff
@@ -6,4 +6,9 @@ just need to get onto one host in a group, and don't care which.
 Usage:
 ```
 rah --inventory PATH_TO_INVENTORY_FILE_OR_DIRECTORY ssh ANSIBLE_GROUP
-```
\ No newline at end of file
+```
+
+Installation:
+```
+pip install https://github.com/simonft/random-ansible-host/zipball/master
```

Comment:
this was just the first solution I found... no idea on the actual
differences between the two

On Tue, Aug 16, 2016 at 2:59 PM Simon Fondrie-Teitler <
notifications@github.com> wrote:

> In README.md
> <https://github.com/simonft/random-ansible-host/pull/1#discussion_r74928870>
> :
>
> > \ No newline at end of file
> > +```
> > +
> > +Installation:
> > +```
> > +pip install https://github.com/simonft/random-ansible-host/zipball/master
>
> Any reason to use this over
>
> pip install git+https://github.com/simonft/random-ansible-host.git?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/simonft/random-ansible-host/pull/1/files/1eea306b17ef15d0e50cec9559b338430ecc99a6#r74928870>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAt8_Ref_Yt11VxLZm9mBIiDCy3a6wBkks5qgbQtgaJpZM4JlWlg>
> .
>